src/scss/tom-select.bootstrap5.scss

```
@@ -1,43 +1,37 @@
 /**
- * Tom Select bootstrap 5
+ * Tom Select Bootstrap 5
  */
 
-//Import Boostrap 5 functions and variables
+// Import Bootstrap 5 functions and variables
 $state-valid: map-get($form-validation-states,'valid') !default;
 $state-invalid: map-get($form-validation-states,'invalid') !default;
-
-
 $enable-shadows: true !default;
 $select-font-family: inherit !default;
 $select-font-size: inherit !default;
-$select-line-height: $input-btn-line-height !default; //formerly line-height-computed
+$select-line-height: $input-btn-line-height !default; // formerly line-height-computed
 
 $select-color-text: $gray-800 !default;
-$select-color-highlight: rgba(255,237,40,0.4) !default;
+$select-color-highlight: rgb(255 237 40 / 40%) !default;
 $select-color-input: $input-bg !default;
 $select-color-input-full: $input-bg !default;
-
-
 $select-color-disabled: $input-disabled-bg !default;
 $select-color-item: #efefef !default;
 $select-color-item-border: $border-color !default;
 $select-color-item-active: $component-active-bg !default;
 $select-color-item-active-text: #fff !default;
-$select-color-item-active-border: rgba(0,0,0,0) !default;
+$select-color-item-active-border: rgb(0 0 0 / 0%) !default;
 $select-color-optgroup: $dropdown-bg !default;
 $select-color-optgroup-text: $dropdown-header-color !default;
 $select-color-optgroup-border: $dropdown-divider-bg !default;
 $select-color-dropdown: $dropdown-bg !default;
-$select-color-dropdown-border-top: mix($input-border-color, $input-bg, 0.8) !default;
+$select-color-dropdown-border-top: mix($input-border-color, $input-bg, 0.8%) !default;
```

```suggestion
$select-color-dropdown-border-top: mix($input-border-color, $input-bg, 80%) !default;
```
Isn't this one also `80%`?

---

I'll look at this and get back to you this afternoon.